### DISCUSSION TITLE
Wallpaper Question\Reuqest

### DISCUSSION THREAD

**Discussion by Evargram** [helloSystem/hello#490] [General]
Hello,

I was wondering if it might be possible to get the default wallpaper for Hello in 4K resolution?

Also I think it would be nice if there was a better method of changing the resolution in Hello.

Thank you.

**Comment by probonopd** [comment]
Hello @Evargram 

> I was wondering if it might be possible to get the default wallpaper for Hello in 4K resolution?

Unfortunately it does not exist in that resolution.

But it is easy enough to change:
Right-click on the desktop, there you can change the desktop picture.

As for the resolution, wasn't your native resolution autodetected properly? Normal users should not need to change the resolution.

But did you try System -> Applications -> Under Construction -> Display?

**Comment by Evargram** [comment]
I was running this in hyper-v to test it out.

Thank you for your reply.

Did you create this wallpaper, or was it downloaded from a different source?

**Reply by probonopd** [reply]
Yes, I had made it some time ago but I don't have it in a higher resolution.

**Reply by Evargram** [reply]
Bravo in it's creation. =)
